ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v3/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -58,6 +58,8 @@ if (typeof window.FileInputWidget === 'undefined') {
             fileNameDom.setAttribute('tabindex', '0');
             fileNameDom.setAttribute('class', "cmp-adaptiveform-fileinput__filename");
             fileNameDom.setAttribute('aria-describedBY', id);
+            fileNameDom.setAttribute('href', fileName);
```

Comment:
Please add test case for this change. Why is this change specific to only v3 ?

Reply:
In v3 span was converted to a tag, for accessibility added href in a tag. In v2 it is span so it is done only in v3